manifests/charts/gateways/istio-egress/templates/_affinity.tpl

```diff
@@ -9,23 +9,25 @@ nodeAffinity:
 {{- end }}
 
 {{- define "nodeAffinityRequiredDuringScheduling" }}
+  {{- $nodeSelector := default .global.defaultNodeSelector .nodeSelector -}}
+  {{- if or .global.arch .nodeSelector }}
```

Comment:
```suggestion
  {{- if or .global.arch $nodeSelector }}
```
I think we need that?

Reply:
Thanks! I thought I tested it. Sorry. Updated and tested again with dummy values.